fractal-puppy/Engine
# Option to Disable skybox + Black Skybox

There was a task to be able to disable the skybox, it was already implemented in the engine. So, this PR improves the disable/enable system and adds another skybox (black).


**To test:**
- Open the engine.
- Go to the Configuration tab.
- Open the Skybox header.
- Disable the "Render Skybox" option and see if the skybox is no longer shown.

**To test 2:**
- Open the engine.
- Go to the Configuration tab.
- Open the Skybox header.
- Change the default skybox for the black one and check if it was changed.
